catalogs/catalog-common/src/main/java/org/apache/gravitino/storage/ABSProperties.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.gravitino.storage;
+
+public class ABSProperties {
+
+  // The account name of the Azure Blob Storage.
+  public static final String GRAVITINO_ABS_ACCOUNT_NAME = "abs-account-name";
+
+  // The account key of the Azure Blob Storage.
+  public static final String GRAVITINO_ABS_ACCOUNT_KEY = "abs-account-key";
```

Comment:
So this is similar to AKSK in Azure blob storage?

Reply:
Yes, Azure block storage also supports other two authentication mechanisms, which are quite complex than the current one.

Reply:
Can you please make sure if this auth is a main stream auth for ABS, and also widely used by other systems?

Reply:
Yes, like AKSK, it is the most widely used by users and application, others like SAS Token and Azure Active Directory (AAD) are quite complicated and hard to configured.